dataprocspawner/spawner.py

```diff
@@ -267,6 +265,11 @@ class DataprocSpawner(Spawner):
       config=True,
       help=""" Allow users to randomize their cluster names. """,)
 
+  internal_ip_only = Bool(
+      False,
```

Comment:
This shouldn't be necessary to have a Hub-level config option. The admins can already ensure that configs available to the user all have the internal ip set.

Reply:
Deleted